tests/views/user_preference_api_tests.py

```diff
@@ -0,0 +1,181 @@
+import json
+import os, uuid
+from django.test import TransactionTestCase
+from django.urls import reverse
+from django.test.utils import captured_stdout
+from django.core import management
+from arches.app.models.models import UserPreference
+from arches.app.models.system_settings import settings
+from arches.app.models import models
+from tests import test_settings
+from django.test.client import Client
+
+
+# these tests can be run from the command line via
+# python manage.py test tests.views.user_preference_api_tests --settings="tests.test_settings"
+
+
+class UserPrefApiTest(TransactionTestCase):
```

Comment:
```suggestion
class UserPrefApiTest(ArchesTestCase):
```
I don't think we need TransactionTestCase. That's a last resort if the functionality under test uses transactions / triggers / anything that would disrupt the django test runner from using transactions to restore state between tests. Spatial views ran triggers.